tools/core_ops.d.ts

```diff
@@ -0,0 +1,287 @@
+import { Buffer } from "node:buffer";
+
+export const op_add_main_module_handler: any;
+export const op_base64_atob: any;
+export const op_base64_btoa: any;
```

Comment:
This will get out-of-sync really quickly with `deno_core` changes. It's probably better than nothing, but I wonder if we can force TSC to use typings coming from `deno_core` package.

Reply:
Fair point. I've reverted that part of the PR to keep the ops untyped for now as I couldn't find an obvious solution to ensure that the ops types don't go out of sync.